packages/teleport/src/config.ts

```diff
@@ -192,6 +197,23 @@ const cfg = {
     return cfg.auth.localAuthEnabled;
   },
 
+  isPasswordlessEnabled() {
+    return cfg.auth.allowPasswordless;
+  },
+
+  getPrimaryAuthType(): PrimaryAuthType {
+    if (
+      cfg.auth.allowPasswordless &&
+      cfg.auth.localConnectorName === 'passwordless'
```

Comment:
Why does passwordless need to be controlled through two separate variables instead of one?

Reply:
i actually don't need to, given below config, if the `connector_name` is set to `passwordless`, disabling passwodless will throw an error as `invalid connector name passwordless disabled`

```
auth_service:
  enabled: yes
  cluster_name: "im-a-cluster-name"
  authentication:
    type: local
    passwordless: true            # enables/disables passwordless
    connector_name: passwordless  # sets passwordless as the default
```